tools/run_local_ironic.sh

```diff
@@ -2,14 +2,37 @@
 
 set -ex
 
-SCRIPTPATH="$(dirname "$(readlink -f "${0}")")"
-
 IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
 IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector"}
 IRONIC_KEEPALIVED_IMAGE=${IRONIC_KEEPALIVED_IMAGE:-"quay.io/metal3-io/keepalived"}
 IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
 IRONIC_DATA_DIR=${IRONIC_DATA_DIR:-"/opt/metal3-dev-env/ironic"}
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+HTTP_PORT="${HTTP_PORT:-6180}"
+PROVISIONING_IP="${PROVISIONING_IP:-172.22.0.2}"
+PROVISIONING_CIDR="${PROVISIONING_CIDR:-24}"
+PROVISIONING_INTERFACE="${PROVISIONING_INTERFACE:-ironicendpoint}"
+DHCP_RANGE="${DHCP_RANGE:-172.22.0.10,172.22.0.100}"
+DEPLOY_KERNEL_URL="${DEPLOY_KERNEL_URL:-http://172.22.0.2:${HTTP_PORT}/images/ironic-python-agent.kernel}"
```

Comment:
Shouldn't the IP address default to `${PROVISIONING_IP}`, not `172.22.0.2`?

In fact, when would you ever need to customise beyond that?

Reply:
Hi, thanks for the comment @zaneb ! I think `${PROVISIONING_IP}` is the same as the `.2` IP as of this 
 [env-file](https://github.com/metal3-io/baremetal-operator/blob/178a7754ea52378a4aae0b66c66bff1865d2ecc8/deploy/ironic_ci.env#L2)?